src/tms_service.ts

```diff
@@ -6,40 +6,10 @@
  */
 
 import _ from 'lodash';
+import { Style as MBStyle, VectorSource as MBVectorSource } from 'maplibre-gl';
```

Comment:
I think we need a new naming convection since you are importing types from maplibre instead of mapbox. How about ML prefix instead of MB prefix? I know this does not match Kibana but we will need to make this change at some point. @thomasneirynck thoughts?

Reply:
> I think we need a new naming convection since you are importing types from maplibre instead of mapbox. How about ML prefix instead of MB prefix? I know this does not match Kibana but we will need to make this change at some point. @thomasneirynck thoughts?

My brain immediately conflates "ML" with  "Machine Learning" rather than "MapLibre". But I agree that we could find a better prefix. Any opposition to "MapLibre", for example "MapLibreStyle"?

Reply:
+1, spelling out to avoid confusion with machine learning makes a lot of sense.